src/Components/LabelInput/LabelInput.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import styles from "./Labelnput.module.scss";
+
+type LabelInputProps = {
+  title: string;
+  name: string;
+  isPassword: boolean;
+  isFirst: boolean;
+  value: string;
+  onChange(args: React.ChangeEvent<HTMLInputElement>): void;
+};
+
+const LabelInput: React.FC<LabelInputProps> = ({
+  title,
+  name,
+  isPassword,
+  isFirst,
+  value,
+  onChange,
+}) => {
+  return (
+    <div className={`${styles.labelInput} ${isFirst ? null : styles.notFirst}`}>
+      <label className={styles.inputTitle} htmlFor={"email"}>
```

Comment:
여기 htmlFor 수정해야 할 것 같습니다!
```suggestion
      <label className={styles.inputTitle} htmlFor={name}>
```

Reply:
수정했습니다